### PR TITLE
Fix invoice formatting bugs

### PR DESCRIPTION
**Pull request by shalapatil**
## Summary
Addressed following bugs in the invoice PDF:
- the address is getting merged with the amount in next section on PDF
- The invoice issue and due date should be displayed in dd-mm-yyyy format
- In the line entry the date should be displayed in dd-mm-yyyy format
- The currency symbol should be displayed instead of USD in amount.
[invoice_c21.pdf](https://github.com/saeloun/miru-web/files/8760708/invoice_c21.pdf)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by supriya3105** [comment]
@shalapatil please resolve the conflicts

File: app/services/invoice_payment/pdf_generation.rb [hunk 1]
```diff
@@ -52,5 +57,31 @@ def format_invoice(invoice_line_items)
...
+        when "JPY"
+          "ja-JP"
+        when "GBP"
+          "en-GB"
+        when "CAD"
+          "en-CA"
+        when "AUD"
+          "en-AU"
+        else
+          "en-US"
```
Comment (rohitjoshixyz) [inline app/services/invoice_payment/pdf_generation.rb:78]: I see similar logic implemented in front-end and is already merged so this looks okay to me

**Comment by supriya3105** [comment]
@apoorv1316 Why hasn't this been test yet ? Please test this and merge is everything is right

**Review by apoorv1316 — APPROVED** [review]

File: Gemfile [hunk 2]
```diff
@@ -184,4 +184,6 @@ gem "searchkick"
 # PDF generator
 gem "grover"
 
+gem "rails-i18n", "~> 7.0.0"
```
Comment (akhilgkrishnan) [inline Gemfile:187]: @shalapatil Why did you add this gem?

**Comment by shalapatil** [comment]
@akhilgkrishnan Removing this gem. Initially I added this gem, because I was settings i18n localization and Money gen was not supporting en-IN.  But yesterday, I removed code for setting i18n localization. Pushing PR for removing the gem.
